apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -19,7 +29,45 @@ import { Apps, AppEvents } from '../../../../ee/server/apps';
 
 const logger = new Logger('RoutingManager');
 
-export const RoutingManager = {
+type Routing = {
```

Comment:
since you are not exporting this type  why not just type the functions implemented on the RoutingManager ?

Reply:
Not quite following, this type is typing the fucntions implemented on routingmanager 🤔 

But, RoutingManager is an object, so the types are "separated". Planned to convert it to a class when we move it to a service.